sql/moz-fx-data-shared-prod/accounts_backend/nonprod_accounts/metadata.yaml

```
@@ -0,0 +1,17 @@
+---
+friendly_name: Accounts table from FxA database in stage environment
+description: |-
+  An authorized view on top of the `accounts_backend_external.nonprod_accounts_v1` table
+  that only includes non-sensitive fields.
+  Some fields in this table are converted to a more user-friendly, BigQuery-native format:
+    - `uid` is converted from bytes to a hex string
+    - `emailVerified` is converted from INTEGER to BOOLEAN
```

Suggest making this more generic, as future boolean columns would presumably also be converted.
```suggestion
    - boolean integer columns are converted to BOOL
```